diff --git a/meson.build b/meson.build
index 6c6c6c6..0000000 100644
--- a/meson.build
+++ b/meson.build
@@ -16,15 +16,7 @@ deps = [xvid_dep]

-incdir = include_directories(
-    run_command(
-        find_program('python', 'python3'),
-        '-c',
-        'import vapoursynth as vs; print(vs.get_include())',
-        check: true,
-    ).stdout().strip(),
-)
+incdir = include_directories()

 py = import('python').find_installation(pure: false)
