From 0ae722f4aaab224f13fe159795748103e4f0335a Mon Sep 17 00:00:00 2001
From: Emily Bowman <silverbacknet@gmail.com>
Date: Mon, 25 Mar 2024 20:46:48 -0700
Subject: [PATCH] Make re strings raw to quiet python warnings

---
 fvsfunc.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fvsfunc.py b/fvsfunc.py
index 765587f..ff29bce 100644
--- a/fvsfunc.py
+++ b/fvsfunc.py
@@ -895,8 +895,8 @@ def ReplaceFrames(clipa, clipb, mappings=None, filename=None):
     # Some people used this as separators and wondered why it wasn't working
     mappings = mappings.replace(',', ' ').replace(':', ' ')
 
-    frames = re.findall('\d+(?!\d*\s*\d*\s*\d*\])', mappings)
-    ranges = re.findall('\[\s*\d+\s+\d+\s*\]', mappings)
+    frames = re.findall(r'\d+(?!\d*\s*\d*\s*\d*\])', mappings)
+    ranges = re.findall(r'\[\s*\d+\s+\d+\s*\]', mappings)
     maps = []
     for range_ in ranges:
         maps.append([int(x) for x in range_.strip('[ ]').split()])
